virtcontainers/pkg/cgroups/manager.go

```diff
@@ -220,7 +207,16 @@ func (m *Manager) moveToParent() error {
 	m.Lock()
 	defer m.Unlock()
 	for _, cgroupPath := range m.mgr.GetPaths() {
-		pids, err := readPids(cgroupPath)
+
+		// first, let's verify this cgroupPath exists:
+		_, err := os.Stat(cgroupPath)
```

Comment:
I think the stat call is technically superfluous/racy in the sense that you can check the error from `m.readPids()` for `os.IsNotExist()`.

Reply:
Good call.  Will give this a try.